spec/acceptance/golden_masters_spec.rb

```diff
@@ -23,11 +23,11 @@
     let(:db) { Sequel.sqlite }
     let(:storage) { RetirementSwap::Storage::Database.new(db) }
     let(:output) { RetirementSwap::Output::IOWriter.new(StringIO.new) }
-    let(:retirement_swap) { RetirementSwap::SwapAlgorithm.new(storage, output) }
+    let(:processor) { RetirementSwap::Processor.new(storage, output, "52c1cf443ae7407d88000001" => {"algorithm" => "swap"}) }
     let(:brokers) { ["kafka:9092"] }
     let(:zookeepers) { ["zk:2181"] }
     let(:topic) { "retirement-swap-test-#{Time.now.to_i}"}
-    let(:reader) { RetirementSwap::Input::KafkaReader.new(processor: retirement_swap,
+    let(:reader) { RetirementSwap::Input::KafkaReader.new(processor: processor,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.